### PR TITLE
FIX: Some minor fixes (cli, high-level server)

### PR DESCRIPTION
**Pull request by klauer**

### CODE REVIEW

File: caproto/_cli.py [hunk 1]
```diff
@@ -344,7 +344,7 @@ def monitor_cli():
                                  "{timedelta} and usages like "
                                  "{timestamp:%%Y-%%m-%%d %%H:%%M:%%S} are "
                                  "supported."))
-    parser.add_argument('-m', type=str, metavar='MASK',
+    parser.add_argument('-m', type=str, metavar='MASK', default='ma',
```
Comment (tacaswell) [inline caproto/_cli.py]: help string says 'va', this says 'ma'

File: caproto/_cli.py [hunk 2]
```diff
@@ -680,7 +680,7 @@ def put(pv_name, data, *, verbose=False, timeout=1, priority=0, repeater=True):
             for command in commands:
                 if (isinstance(command, ca.WriteNotifyResponse) and
                             command.ioid == req.ioid):
-                    response = command
+                    # response = command
```
Comment (tacaswell) [inline caproto/_cli.py]: why?  Looks like it isn't used?

File: caproto/_dbr.py [hunk 3]
```diff
@@ -20,11 +20,10 @@
     USE_NUMPY = True
 
 
-class AccessRights(IntEnum):
+class AccessRights(IntFlag):
```
Comment (tacaswell) [inline caproto/_dbr.py]: nifty! https://docs.python.org/3/library/enum.html#intflag

3.6 is paying divides already.

File: caproto/_cli.py (hunk 2 above)
Comment (klauer) [reply caproto/_cli.py]: @danielballan ?

Comment (danielballan) [reply caproto/_cli.py]: Relic of a copy/paste I think. Safe to delete.

**Pull request merged**